### ISSUE TITLE
Allow for screen reader support with regards to authorship information

### ISSUE THREAD

**Issue by nidico** [hedgedoc/hedgedoc#2896]
[accessibility]

## Problem:

In order to allow screen readers to read text with individual voices or announce authorship changes, authorship information needs to be properly and (preferably) **consistently** marked up.

For some reason, authorship information is shown in two different ways in CodiMD (colored vertical bars and underlines) (see hedgedoc/hedgedoc#2892 for a discussion and possible enhancements).

Due to that and in particular due to hedgedoc/hedgedoc#2895, screen readers cannot (neither easily nor at all) determine authorship.

## Solution:

Add `authorship-inline-xxx` classes for all (explicitly authored) content (as done in hedgedoc/hedgedoc#2892, "simple proposal", but not necessarily visible).

## Related issues:

- hedgedoc/hedgedoc#2892: Use of vertical bars in gutter vs inline underline for authorship information display
- codemirror/CodeMirror#4604: Accessibility for blind and visually impaired screen reader users. According to this issue, [CodeMirror 6](https://codemirror.net/6/) will be much better accessibility-wise.
- hedgedoc/hedgedoc#2894: Highlight authorship in a color independent way
